base/range.jl

```diff
@@ -66,8 +69,12 @@ Mathematically a range is uniquely determined by any three of `start`, `step`, `
 Valid invocations of range are:
 * Call `range` with any three of `start`, `step`, `stop`, `length`.
 * Call `range` with two of `start`, `stop`, `length`. In this case `step` will be assumed
-  to be one. If both arguments are Integers, a [`UnitRange`](@ref) will be returned.
-* Call `range` with one of `stop` or `length`. `start` and `step` will be assumed to be one.
+  to be positive one. If both arguments are Integers, a [`UnitRange`](@ref) will be returned.
```

Comment:
I find the phrasing "positive one" a bit confusing. Why not just say "one", as it used to be? Or perhaps `1`?

Reply:
To emphasize that it is not a magnitude, which seems to be the frame a lot of people come in with.

Reply:
Seems reasonable to me, and I don't feel strongly about this to belabor the point. Thanks!